src/components/Select/Select.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+export interface Options {
+  label: string
+  value: string
+}
+
+export interface Props {
+  className?: string
+  value: string
+  name: string
+  required?: boolean
+  disabled?: boolean
+  error?: boolean
+  width?: number | string
+  options: Options[]
+  blankOption?: boolean
+  placeholder?: string
+  onChange?: (name: string, value: string) => void
+}
+
+interface StyledProps extends InjectedProps {
+  width: string
+}
+
+interface SelectEvent {
+  currentTarget: {
+    value: string
+  }
+}
+
+class SelectComponent extends React.PureComponent<Props & InjectedProps> {
+  public render() {
+    const {
+      className = '',
+      value = '',
+      name,
+      required = false,
+      disabled = false,
+      error = false,
+      width = 260,
+      theme,
+      options,
+      blankOption = false,
+      placeholder = '',
+    } = this.props
+    const widthStyle = typeof width === 'number' ? `${width}px` : width
+    const classNames = `${className} ${error ? 'error' : ''}`
+
+    return (
+      <Wrapper width={widthStyle} theme={theme}>
+        <Base
+          className={classNames}
+          value={value}
+          name={name}
+          required={required}
+          disabled={disabled}
+          onChange={this.handleChange}
+          theme={theme}
+        >
+          {blankOption && <option value=""> {placeholder}</option>}
```

Comment:
[ask] Is there a pattern where blankOption is true and there is no placeholder? 
How about putting out elements by looking at placeholder values?

Reply:
I think `blackOption` and `placeholder ` are unnecessary for this component.
The user can do that same by simply passing `{label: 'Fruit', value: '' }`

Reply:
> The user can do that same by simply passing {label: 'Fruit', value: '' }

That's true.
BlackOption and placeholder are unnecessary.

Fixed here.
https://github.com/kufu/smarthr-ui/pull/356/commits/6d7bc5dca026fb0163f65bec0dd9675d83b3e780